src/Sarif.UnitTests/SarifTransformerTests.cs

```diff
@@ -365,5 +366,84 @@ public void SarifTransformerTests_ToCurrent_OneRunWitRules()
             rule.MessageStrings.Should().BeNull();
             rule.Properties.Should().BeNull();
         }
+
+        [Fact]
+        public void SarifTransformerTests_ToCurrent_OneRunWithBasicInvocation()
+        {
+            string v1LogText =
+              @"{
+                  ""version"": ""1.0.0"",
+                  ""runs"": [
+                    {
+                      ""invocation"": {
+                        ""commandLine"": ""CodeScanner @collections.rsp"",
+                        ""responseFiles"": {
+                          ""collections.rsp"": ""-input src/collections/*.cpp -log out/collections.sarif -rules all -disable C9999""
```

Comment:
This isn't right: `responseFiles` is now an array of `fileLocation` objects, each of which specifies the location of a response file. To embed the response file contents, you make a corresponding entry in the `run.files` dictionary, and populate `file.contents.text`.

I'm surprise this even works... oh, I see what you did! You abused `uriBaseId` to hold the contents. Umm, no :-) :-)

Reply:
(Actually, although my diagnosis turns out to be right, my comment above was premature. I forgot that this is a v1 file :-))

Reply:
```
{                        # A run object
  "files": {
    "collections.rsp": {
      "contents": {
        "text": "-input src/collections/*.cpp -log out/collections.sarif -rules all -disable C9999"
      }
    }
  }

  "invocations": [
    {
      "responseFiles": [
        {
          "uri": "collections.rsp"
        }
      ]
    }
  ]
}

Reply:
Well this json is the v1 version which I took from Comprehensive.sarif. In the transfer coe, I did use UriBaseId for the contents based on my interpretation of this statement in 3.13.4:
A SARIF producer MAY embed the contents of a response file in the SARIF log file by mentioning the response file in run.files (§3.11.11), and providing a value for the file.contents property (§3.16.8).
So I have apparently misinterpreted that!

Reply:
The "MAY" means that you don't have to embed the contents at all. But if you do, the go in `file.contents.text` as in my example. No worries.

Reply:
Gotcha